src/components/Register.js

```diff
@@ -1,58 +1,82 @@
-								type="email"
-								className="form-control"
-								id="email"
-								autoComplete="email"
-								placeholder="Email"
-							/>
-						</div>
-						<hr></hr>
-						<h3>Create a Password</h3>
-						<div className="form-group">
-							<input
-								type="password"
-								className="form-control"
-								id="password"
-								autoComplete="current-password"
-								placeholder="Password"
-							/>
-						</div>
-						<button type="submit" className="btn btn-primary">
-							Submit
-						</button>
-					</div>
-				</form>
-			</div>
-		);
-	}
+class Register extends Component {
+  constructor() {
+    super();
+    this.state = {
+      password: '',
+      email: '',
+      first_name: '',
+      last_name: '',
+	    phone: '',
+
+    };
+    this.handleChange = this.handleChange.bind(this);
+    this.handleSubmit = this.handleSubmit.bind(this);
+  }
+
+  handleChange(ev) {
+    this.setState({      
+      [ev.target.name]: ev.target.value,
+    });
+  }
+
+  async handleSubmit(ev) {
+	ev.preventDefault();
+	await this.props.signUp({...this.state})
+	
+    this.setState({
+      password: '',
+      email: '',
+      first_name: '',
+      last_name: '',
+	    phone: '',
+	});	
+  }
+
+  render() {
+	const { handleChange, handleSubmit } = this;
+    return (
+      <div>
+        <form style={userStyles} onSubmit={handleSubmit}>
+          <div style={{ margin: '20px' }}>
+            <h3>Personal Info</h3>
+            <div className="form-group">
+              <input type="text" className="form-control" id="firstName" placeholder="First Name" name="first_name" onChange={handleChange} value={this.state.first_name } />
+            </div>
+            <div className="form-group">
+              <input type="text" className="form-control" id="lastName" placeholder="Last Name" name="last_name" onChange={handleChange} value={this.state.last_name} />
+            </div>
+            <hr />
+            <h3>Contact Info</h3>
+            <div className="form-group">
+              <input type="text" className="form-control" id="phoneNumber" placeholder="Phone Number" name="phone" onChange={handleChange} value={this.state.phone} />
+            </div>
+            <div className="form-group">
+              <input type="email" className="form-control" id="email" autoComplete="email" placeholder="Email" name="email" onChange={handleChange} value={this.state.email} />
+            </div>
+            <hr />
+            <h3>Create a Password</h3>
+            <div className="form-group">
+              <input type="password" className="form-control" id="password" autoComplete="current-password" laceholder="Password" name="password" onChange={handleChange} value={this.state.password} />
+            </div>
+            <button type="submit" className="btn btn-primary"> Submit </button>
+          </div>
+        </form>
+      </div>
+    );
+  }
 }
+
+export default connect(
+  ({ user }) => ({
+    user: user.user,
```

Comment:
![Screen Shot 2020-12-04 at 2 26 01 PM](https://user-images.githubusercontent.com/8497366/101206303-f0c03c80-363c-11eb-9772-abba93744676.png)

I was getting an error on this line. 
After changing it to:

`user: user.user,`

the error went way.